node/src/reactor/main_reactor/event.rs

```diff
@@ -311,7 +311,7 @@ impl ReactorEvent for MainEvent {
             MainEvent::ChainspecRawBytesRequest(_) => "ChainspecRawBytesRequest",
             MainEvent::UpgradeWatcherRequest(_) => "UpgradeWatcherRequest",
             MainEvent::StorageRequest(_) => "StorageRequest",
-            MainEvent::BlockCompleteConfirmationRequest(_) => "MarkBlockCompletedRequest",
+            MainEvent::BlockCompleteConfirmationRequest(_) => "BlockCompleteConfirmationRequest",
```

Comment:
Although it's not in the scope of this PR, I don't like the name discrepancy here. `BlockCompleteConfirmationRequest` and `MarkBlockCompletedRequest` seem to be used interchangeably in the codebase, and they don't seem to mean the same to me as far as naming goes. I checked and it seems it's the storage request for marking blocks complete, but `BlockCompleteConfirmationRequest` sounds like you'd ask storage if this block is complete. Again, not that relevant to this PR, just putting this out there.

Reply:
I agree

Reply:
It'll be addressed here: https://github.com/casper-network/casper-node/pull/3868